Data/CubeBlocks.sbc

```diff
@@ -27,20 +27,15 @@
         <Component Subtype="SteelPlate" Count="40" />
       </Components>
       <CriticalComponent Subtype="RadioCommunication" Index="0" />
-      <MountPoints>
-        <MountPoint Side="Bottom" StartX="0.1" StartY="0.1" EndX="0.9" EndY="0.9" />
-        <MountPoint Side="Top" StartX="0.1" StartY="0.1" EndX="0.9" EndY="0.9" />
-        <MountPoint Side="Left" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-        <MountPoint Side="Right" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-        <MountPoint Side="Front" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-        <MountPoint Side="Back" StartX="0.175" StartY="0.1" EndX="0.825" EndY="0.9" />
-      </MountPoints>
-      <BuildProgressModels>
-        <Model BuildPercentUpperBound="0.33" File="Models\Cubes\Large\BeaconConstruction_1.mwm" />
-        <Model BuildPercentUpperBound="0.66" File="Models\Cubes\Large\BeaconConstruction_2.mwm" />
-        <Model BuildPercentUpperBound="1.00" File="Models\Cubes\Large\BeaconConstruction_3.mwm" />
-      </BuildProgressModels>
-      <BlockPairName>GardenConquest_UnlicensedHullClassifier</BlockPairName>
+			<BuildProgressModels>
+				<Model BuildPercentUpperBound="0.33" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr1.mwm" />
+				<Model BuildPercentUpperBound="0.67" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr2.mwm" />
+				<Model BuildPercentUpperBound="1.00" File="Models\Cubes\Unlicensed 1\GCUnlicensedHullClassifier_Constr3.mwm" />
+			</BuildProgressModels>
+			<MountPoints>
+				<MountPoint Side="Bottom" StartX="0.03" StartY="0.03" EndX="0.97" EndY="0.97" />
```

Comment:
I know nothing about mountpoints, so hopefully you can enlighten me here. What's the reason for using a different Start and End attributes (`.03` and `.97` vs `.1` and `.9`) for the bottom mountpoint? And does anyone think removing the other mountpoints from the sides of the base is too restrictive?